tests/test_collector.py

```diff
@@ -0,0 +1,109 @@
+from http import HTTPStatus
+
+import pytest
+from gitlab import Gitlab
+from gitlab.exceptions import GitlabGetError
+from prometheus_client.core import GaugeMetricFamily
+from pytest_mock import MockerFixture
+
+from gitlab_project_exporter.collector import (
+    GitLabProjectCollector,
+    RemoteMirrorCollectionResult,
+)
+from gitlab_project_exporter.gitlab_project import GitlabProject
+
+from .conftest import (
```

Comment:
Same as in `test_api`

Reply:
fixed in https://github.com/app-sre/gitlab-project-exporter/commit/baf4ac06dd3c452d48f7310d4c34f14727c1f4c5